src/Microsoft.Sbom.Api/Executors/IJsonSerializationStrategy.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+public interface IJsonSerializationStrategy
```

Comment:
Does the interface need to be public? I see no public access to it. Let's make it `internal` if we can

Reply:
This is tested in the workflow tests where there are UTs to switch to different serialization strategies based on the manifest version. So I don't think that there's necessarily a gap here.